src/hooks/useDarkMode.js

```diff
@@ -0,0 +1,10 @@
+import useLocalStorage from "./useLocalStorage";
+
+const useDarkMode = (key, initialValue) => {
+  const [isDarkMode, setDarkMode] = useLocalStorage(key, initialValue);
+  const toggleDarkMode = () => {
+    setDarkMode(isDarkMode);
```

Comment:
daca faci `setDarkMode(isDarkMode);`, practic nu setezi nimic, asta cum ai face `setState(true)`, deci intotdeauna ii dai aceeasi valoare

Reply:
atunci nu inteleg pentru ce am nevoie de acesta functie??
_**Finally, we need to return something out of useDarkMode, so we can use this in our app. What do you think we'll need? We'll need to know if dark mode is enabled, right? And we'll need a setter function to toggle dark mode. Let's just forward the value and the setter that were returned out of the useLocalStorage call. Return those two values in an array as well._**

Reply:
deci ai nevoie de o functie care transforma `darkMode` din `true` in `false`, respectiv tu vrei sa faci `setDarkMode(!isDarkMode);` pentru a-i da intotdeauna inversul